damus.xcodeproj/project.pbxproj

```diff
@@ -421,6 +431,7 @@
 		4C75EFA227FA576C0006080F /* Views */ = {
 			isa = PBXGroup;
 			children = (
+				3169CAE4294E699400EE4006 /* Empty Views */,
```

Comment:
I see this change here does not have .swift file-extension unlike the rest below.

Reply:
Yeah this is just a folder. Although good catch cuz I actually wanted to get rid of this.